libvmi/driver/bareflank/bareflank.c

```diff
@@ -68,7 +68,7 @@ static inline uint64_t parse_reg_value(const char *reg, json_object *root)
     // parse the json and get the value of the key
     json_object *return_obj = NULL;
     json_object_object_get_ex(root,reg,&return_obj);
-    return json_object_get_int64(return_obj);
+    return (uint64_t) json_object_get_uint64(return_obj);
```

Comment:
This is a new API introduced in **json-c 0.14** allowing us to not have to split `uin64_t` values into high/low values anymore.

Reply:
Well, if you rely on a particular API being available you better check for that during the configure stage to alert the user that they need to install a newer version of the lib. Especially if its not something available by default in the repository.